src/deploy/NetworkConfiguration.ts

```diff
@@ -178,7 +186,7 @@ export async function getConfiguration(
   let contractMap = contractMapOverride ?? await deploymentManager.contracts();
 
   let symbol = networkConfiguration.symbol;
-  let baseToken = getContractAddress(networkConfiguration.baseToken, contractMap);
+  let baseToken = hasContractAddress(networkConfiguration.baseToken, contractMap) ? getContractAddress(networkConfiguration.baseToken, contractMap) : networkConfiguration.baseTokenAddress;
```

Comment:
Should we just add a third fallback/default arg to `getContractAddress`?

Reply:
Yeah that's cleaner. I added a third arg to `getContractAddress`.